composer.json

```diff
@@ -20,11 +20,17 @@
         "yoast/phpunit-polyfills": "^2.0"
     },
     "scripts": {
-        "lint": "@phpcs",
-        "phpcs": "vendor/bin/phpcs",
-        "phpcbf": "vendor/bin/phpcbf",
-        "phpunit": "vendor/bin/phpunit",
-        "test": "@phpunit"
+        "lint": [
+            "@phpcs",
+            "@phplint"
+        ],
+        "phpcs": "vendor/bin/phpcs --ignore=tests/* -s .",
+        "phpcbf": "vendor/bin/phpcbf --ignore=tests/* .",
```

Comment:
optional.
@jazzsequence and I disagree on this, but I believe tests should be linted when we can. Not worth a messy change if they never have been, but if they've been included in linting until now, don't stop.